ui/core/src/model/units/units.ts

```diff
@@ -47,21 +48,24 @@ export const UNIT_GROUP_CONFIG: Readonly<Record<UnitGroup, UnitGroupConfig>> = {
   Decimal: DECIMAL_GROUP_CONFIG,
   Bytes: BYTES_GROUP_CONFIG,
   Throughput: THROUGHPUT_GROUP_CONFIG,
+  Currency: CURRENCY_GROUP_CONFIG,
 };
 export const UNIT_CONFIG = {
   ...TIME_UNIT_CONFIG,
   ...PERCENT_UNIT_CONFIG,
   ...DECIMAL_UNIT_CONFIG,
   ...BYTES_UNIT_CONFIG,
   ...THROUGHPUT_UNIT_CONFIG,
+  ...CURRENCY_UNIT_CONFIG,
 } as const;
 
 export type FormatOptions =
   | TimeFormatOptions
   | PercentFormatOptions
   | DecimalFormatOptions
   | BytesFormatOptions
-  | ThroughputFormatOptions;
+  | ThroughputFormatOptions
+  | CurrencyFormatOptions;
```

Comment:
this should be replicated on the CUE side as well https://github.com/perses/perses/blob/main/cue/common/format.cue#L16